components/service/experiments/src/main/java/mozilla/components/service/experiments/Experiments.kt

```diff
@@ -87,12 +93,29 @@ open class ExperimentsInternalAPI internal constructor() {
         evaluator = ExperimentEvaluator(ValuesProvider())
         valuesProvider = ValuesProvider()
 
-        source = null
+        updater?.let {
+            it.source = null
+        }
         storage = null
 
         isInitialized = false
     }
 
+    internal fun getGlean(): GleanInternalAPI {
+        return Glean
+    }
+
+    internal open fun getExperimentsUpdater(context: Context): ExperimentsUpdater {
+        return ExperimentsUpdater(context, this)
+    }
+
+    internal open fun getExperimentsStorage(context: Context): FlatFileExperimentStorage {
```

Comment:
Why is this function `open`? I can't seem to find a place in this PR in which it is overriden. Are you planning a follow-up PR for using this?